pkg/cluster/sync.go

```diff
@@ -24,6 +24,7 @@ func (c *Cluster) Sync(newSpec *spec.Postgresql) (err error) {
 
 	defer func() {
 		if err != nil {
+			c.logger.Warningf("error while syncing cluster state: %v", err)
```

Comment:
Is this somehow related to the subject of PR, or just a random improvement?

Reply:
General +1 for random improvements regardless of the answer to the previous question.

Reply:
This was a random improvement ensuring we do not silently swallow an error.